packages/core/src/render3/i18n/i18n_parse.ts

```diff
@@ -604,12 +604,12 @@ export function i18nParseTextIntoPartsAndICU(pattern: string): (string|IcuExpres
 export function parseIcuCase(
     tView: TView, tIcu: TIcu, lView: LView, updateOpCodes: I18nUpdateOpCodes, parentIdx: number,
     caseName: string, unsafeCaseHtml: string, nestedIcus: IcuExpression[]): number {
-  const create: I18nMutateOpCodes = [];
-  const remove: I18nMutateOpCodes = [];
-  const update: I18nUpdateOpCodes = [];
+  const create: IcuCreateOpCodes = [] as any;
+  const remove: I18nRemoveOpCodes = [] as any;
+  const update: I18nUpdateOpCodes = [] as any;
```

Comment:
NIT you can just do:

```suggestion
  const create = [] as IcuCreateOpCodes;
  const remove = [] as I18nRemoveOpCodes;
  const update = [] as I18nUpdateOpCodes;
```

Reply:
nope, that does not work, because of the brand.

Reply:
OH! Sorry. I didn't realise.